pandas/core/arrays/sparse/accessor.py

```diff
@@ -329,7 +329,7 @@ def to_coo(self):
         import_optional_dependency("scipy")
         from scipy.sparse import coo_matrix
 
-        dtype = find_common_type(self._parent.dtypes)
+        dtype = find_common_type(self._parent.dtypes.to_list())
```

Comment:
this is because `find_commont_type` from `pandas/core/dtypes/cast.py` requires a list